client/analytics/components/leaderboard/test/__snapshots__/index.js.snap

```diff
@@ -46,7 +46,7 @@ exports[`Leaderboard should render correct data in the table 1`] = `
                 </span>
               </th>
               <th
-                class="woocommerce-table__header"
+                class="woocommerce-table__header is-left-aligned"
```

Comment:
I would love to remove these tests in favor of testable expectations, even if that's a single test around the alignment for now.  Do you have the bandwidth to tackle this?

Reply:
@joshuatf  - thanks for your review! 

I am interested in tackling this! 

> I would love to remove these tests in favor of testable expectations, even if that's a single test around the alignment for now. 

Do you mean I should add a test for this component `Leaderboard`, or a test for my changes in component `Table`?

IMHO: 

- It makes more sense to test the alignment for `Table`.
- For `Leaderboard`, we may still keep the JS snap or update our current code to match with the previous snapshot? The main reason is that two current tests in client/analytics/components/leaderboard/test/index.js relies on snapshot. I can make changes but that will lose focus of this PR.

Reply:
@htdat thanks for wanting to tackle this. I will just follow up on your question for @joshuatf (and he can correct me if I am wrong 😄 )

> Do you mean I should add a test for this component Leaderboard, or a test for my changes in component Table?

Add a test for the `Table` component in [it's test file](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/components/src/table/test/index.js).
I would say two or three test cases to see if the `is-left-aligned` class is present, when `isNumeric` is either false or true.

You can leave the `Leaderboard` test changes as is, we are planning on removing all the snapshot tests at some point, removing it from `Leaderboard` will be on our todo list.

Reply:
@louwie17 thank you, it's clear now 👍 . I've added tests in 9f4a656